bdsx/bds/player.ts

```diff
@@ -958,6 +958,68 @@ export class SimulatedPlayer extends ServerPlayer{
     simulateDisconnect(): void{
         abstract();
     }
+    simulateLookAtBlock(targetBlock:BlockPos):void{
+        abstract();
+    }
+    simulateLookAtEntity(target:Actor):void{
+        abstract();
+    }
+    simulateLookAtLocation(targetPos:Vec3):void{
+        abstract();
+    }
+    simulateJump():void{
+        abstract();
+    }
+    simulateSetBodyRotation(rotation:number):void{
+        abstract();
+    }
+    simulateSetItem(item:ItemStack,bool:boolean,int:number):boolean{
```

Comment:
`bool` should be `shouldSelectSlot` and `int` is `slot`

Reply:
I see
I'll commit them.